docs/how-to/add-model-and-api-endpoints.md

```diff
@@ -25,91 +71,195 @@ https://github.com/fyliu/peopledepot/blob/acd8898e7b0364913cc8ae3f9973dfd846aded
+   1. [list_filter](https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_filter) adds filter controls to declared fields (useful, but not shown in this example).
 
-### View the admin site to see everything's working and there are no issues, which should be fine unless there's custom input fields
+### View the admin site to see that everything's working and there are no issues, which should be the case unless there's custom input fields creating problems
 
-1. See the [contributing doc](https://github.com/fyliu/peopledepot/blob/development/docs/contributing.md#:~:text=Browse%20to%20the%20web%20admin%20interface%20at%20http%3A//localhost%3A8000/admin/) for how to view the admin interface.
+1. See the [contributing doc](https://github.com/hackforla/peopledepot/blob/main/docs/CONTRIBUTING.md#:~:text=Browse%20to%20the%20web%20admin%20interface%20at%20http%3A//localhost%3A8000/admin/) for how to view the admin interface.
 
-1. Example of a custom field
+1. Example of a custom field (as opposed to the built-in ones)
+
+   ```python
+   time_zone = TimeZoneField(blank=True, use_pytz=False, default="America/Los_Angeles")
+   ```
 
-   https://github.com/fyliu/peopledepot/blob/acd8898e7b0364913cc8ae3f9973dfd846adedcc/app/core/models.py#L95
+   [link to code](https://github.com/hackforla/peopledepot/blob/09e2856b6dd8038aedbbc9b42c3a44009be1fd2f/app/core/models.py#L96)
 
-   1. Having this field could cause the admin site to crash and the developer will need to look at the debug message and resolve it
+   1. Having a misconfigured or buggy custom field could cause the admin site to crash and the developer will need to look at the debug message and resolve it.
 
 ### Tests
 
 1. Feel free to write tests for the admin. There's no example for it yet.
-1. The reason there's no tests is that the admin site is independent of the API functionality, and we're mostly interested in the API part.
+1. The reason there's no tests is that the admin site is independent of the API functionality, and we're mainly interested in the API part.
+1. When the time comes that we depend on the admin interface, we will need to have tests for the needed functionalities.
 
-## Work on the API endpoints
+### Check point 2
+
+This is a good place to pause, check, and save. Refer to the [contributing doc](https://github.com/hackforla/peopledepot/blob/main/docs/CONTRIBUTING.md) sections 3.4-3.9 for details.
+
+1. Run pre-commit checks
+
+   ```bash
+   ./scripts/precommit-check.sh
+   ```
 
-There's several steps for adding API endpoints.
+1. Add and commit changes
+
+   ```bash
+   git add -A
+   git commit -m "feat: register admin: recurring_event"
+   ```
+
+## The API
+
+There's several components to adding API endpoints: Model(already done), Serializer, View, and Router.
```

Comment:
I think it might be nice to add this on line 22 as an overview?

There's several components to adding an API endpoint: Model, Serializer, View, and Router.

Reply:
Not really for this one. The model is just the prerequisite and the other items are part of the API steps.

Reply:
There's a short sentence in the intro before the "Let's begin" that mentions the 3 parts.